sdks/java/core/src/test/java/org/apache/beam/sdk/util/BigQueryUtilTest.java

```diff
@@ -81,6 +83,7 @@
   @Mock private Bigquery.Tables.Get mockTablesGet;
   @Mock private Bigquery.Tabledata mockTabledata;
   @Mock private Bigquery.Tabledata.List mockTabledataList;
+  private PipelineOptions options = PipelineOptionsFactory.create();
```

Comment:
Create a new instance in setUp() to prevent changes from propagating between tests